### ISSUE TITLE
Grid does not update on reordering of grid items (ie via ui-sortable)

### ISSUE THREAD

**Issue by robertjchristian** [angular-ui/ui-grid#272]
The model changes, but the grid does not update.  It does update on splice.  So as a workaround I can sort (change ordering of elements within model) then within the grid, click a button (delete of an arbitrary element) that results in a splice call, and at that point the new order is minded.

As an attempt at working around, also tried reassignment by stringifying json and marshalling it back into an object - that does not trigger an update either.

In a nutshell the only way I can get the grid to mind an update to the model is to enact the splice function.
